CorsixTH/Lua/objects/reception_desk.lua

```diff
@@ -92,48 +92,74 @@ function ReceptionDesk:onClick(ui, button)
 end
 
 function ReceptionDesk:tick()
+  --! Handle a patient at the reception desk.
+  --!param patient The patient at the front of queue.
+  --!param new (bool) Have they just entered the hospital?
+  local function handlePatient(patient, new)
+    if not new then
+      -- Patient redirected to reception, send them on their way
+      local id = patient.next_room_to_visit.room_info.id
+      return patient:queueAction(SeekRoomAction(id))
+    end
+
+    --else: a new patient has arrived in the hospital
+    if patient:agreesToPay("diag_gp") then
+      patient:queueAction(SeekRoomAction("gp"))
+    else
+      patient:goHome("over_priced", "diag_gp")
+    end
+  end
+
+  -- Sends the VIP on their way.
+  local function handleVIP(vip)
+    vip:queueAction(IdleAction())
+    vip.waiting = 1
+  end
+
+  -- Let the Inspector conclude the epidemic, then send them on their way.
+  local function handleInspector(inspector)
+    if inspector.going_home then return end
+    local epidemic = self.hospital.epidemic
+    if epidemic then
+      -- The result of the epidemic may already by determined
+      -- i.e if an infected patient has left the hospital
+      if not epidemic.result_determined then
+        epidemic:finishCoverUp()
+      end
+      epidemic:applyOutcome()
+      inspector:goHome()
+    end
+  end
+
   local queue_front = self.queue:front()
-  local reset_timer = true
-  if self.receptionist and queue_front then
+  --! Advance the front humanoid of the queue.
+  --! Returns true if ready to be dealt with.
```

Comment:
```suggestion
  --!return (boolean) Returns true if ready to be dealt with, else false.
```
In this way ldoc understands that the second sentence is about the return value.